### ISSUE TITLE
If rootUrl contains path tests don't work.

### ISSUE THREAD

**Issue by sergesemashko** [bem/gemini#112]
Works:
```
# .gemini.yml : 
rootUrl: http://local.mysite.com
```
```js
// testSuite.js:
var gemini = require('gemini');
gemini.suite('media-gallery', function (suite) {
  suite.setUrl('/patternlab/media-gallery-page.html')
...
```

Doesn't work:
```
# .gemini.yml : 
rootUrl: http://local.mysite.com/patternlab
```
```js
// testSuite.js:
var gemini = require('gemini');
gemini.suite('media-gallery', function (suite) {
  suite.setUrl('/media-gallery-page.html')
...
```

I'm getting following error using phantomJs as a webdriver:
```bash
✘ media-gallery plain [phantomjs]
StateError: Can not find element with css selector: .my-selector
```

**Comment by SevInf** [comment]
This is happening because of `url.resolve` node function which we are using for computing absolute URLs. We are going to replace it with something that will work as expected in this scenario, but for now you can use `rootUrl: http://local.mysite.com/patternlab/` (with a slash at the end) to workaround this problem.

**Comment by sergesemashko** [comment]
got it, thanks for a workaround

**Comment by SevInf** [comment]
Fixed in #144
